include/mbedtls/ssl.h

```diff
@@ -4739,6 +4748,11 @@ int mbedtls_ssl_get_session(const mbedtls_ssl_context *ssl,
  * \return         #MBEDTLS_ERR_SSL_HELLO_VERIFY_REQUIRED if DTLS is in use
  *                 and the client did not demonstrate reachability yet - in
  *                 this case you must stop using the context (see below).
+ * \return         #MBEDTLS_ERR_SSL_RECEIVED_EARLY_DATA if early data, as
+ *                 defined in RFC 8446 (TLS 1.3 specification), has been
+ *                 received as part of the handshake. This is server specific.
+ *                 You must call mbedtls_ssl_read_early_data() to read the
+ *                 early data before to resume the handshake.
```

Comment:
Language: I think it should be "before resuming". (Multiple occurrences in this file.)